designcompose/src/main/java/com/android/designcompose/squoosh/SquooshText.kt

```diff
@@ -150,30 +146,27 @@ internal fun squooshComputeTextInfo(
             } else if (v.data.hasStyledText()) {
                 val builder = AnnotatedString.Builder()
                 for (run in getTextContent(appContext, v.data.styledText)) {
-                    val style = run.styleOrNull.getOrThrow("StyledTextRun")
+                    val style = run.style
```

Comment:
I'd probably keep these around for a little while, and delete them later once we are stable.

Reply:
What about replacing them with !!? Does basically the same thing and I don't need to restore those functions.

Reply:
I'm ok trying that. If it gets hit a lot I'd consider putting the function back in.